cristin-import/src/main/java/no/unit/nva/cristin/CristinMapper.java

```diff
@@ -0,0 +1,81 @@
+package no.unit.nva.cristin;
+
+import java.net.URI;
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Stream;
+import no.unit.nva.model.AdditionalIdentifier;
+import no.unit.nva.model.EntityDescription;
+import no.unit.nva.model.Publication;
+import no.unit.nva.model.PublicationDate;
+
+public class CristinMapper {
+
+    private final CristinObject cristinObject;
+
+    public CristinMapper(CristinObject cristinObject) {
+        this.cristinObject = cristinObject;
+    }
+
+    public Publication generatePublication() {
+        return new Publication.Builder()
+                   .withAdditionalIdentifiers(Set.of(extractIdentifier()))
+                   .withEntityDescription(generateEntityDescription())
+                   .withCreatedDate(extractEntryCreationDate())
```

Comment:
Some initial conversions. It is not certain if they will remain. I am mostly interested in the conversion pattern